### DISCUSSION TITLE
Need a hand for 'unet_patcher'

### DISCUSSION THREAD

**Discussion by ChrisGitHubIcon** [lllyasviel/stable-diffusion-webui-forge#2249] [Q&A]
I am facing a problem of module_forge.unet_patcher is missing, which can not activate AnimateDiff
my system is Debian CLI and git clone from stable_diffusion_webui_forge/master
Is ther anyone having same issue? 
Or anyone can pull me out, highly appreciate

**Comment by s4130** [comment]
如果你要使用animatediff可以使用旧版本的forge或者reforge

**Reply by ChrisGitHubIcon** [reply]
我是远程部署在服务器上，用的CLI，而且翻墙的原因，每次重装，下forge和pytorch和各种modols，再debug都是很痛苦的过程。。。
不知道在现有的版本下能不能解决这个问题？
我去继续革命那边也留了言，不知道什么时候这两个能sync到一起。。。
